mailpoet/assets/js/src/newsletters/send.tsx

```diff
@@ -460,8 +480,8 @@ class NewsletterSendComponent extends Component<
           ) {
             this.context.notices.success(
               <p>
-                {MailPoet.I18n.t('automaticEmailActivated').replace(
-                  '%1s',
+                {sprintf(
```

Comment:
I could have left it like that, but since this `sprintf` is provided by the `@wordpress/i18n` package and mentioned in examples, I wanted to give it a try and mention it here, also a ticket is created to adjust our current translations with custom placeholders and start using `sprintf`, this will enable us to get rid of https://www.npmjs.com/package/react-string-replace from our dependencies and start using [a package  that's already installed](https://www.npmjs.com/package/sprintf-js)

Reply:
`sprintf` is a standard companion of `__` in the `@wordpress/i18n` library, and it's also used in Gutenberg, so the migration makes sense. If it's systematic, replacing all `.replace` calls.

Reply:
Yeah, we could actually create a ticket to adopt `sprintf` everywhere. But maybe it makes even more sense to accompany it with the adoption of [@wordpress/element](https://github.com/WordPress/gutenberg/blob/trunk/packages/element/README.md) and using [createInterpolateElement](https://github.com/WordPress/gutenberg/blob/trunk/packages/element/README.md#createinterpolateelement) instead of [react-string-replace](https://github.com/iansinnott/react-string-replace). Or we can create two separate tickets.

Reply:
I've already [created](https://mailpoet.atlassian.net/browse/MAILPOET-5117) a ticket for such cases , it entails changing the placeholders in the translatable strings, other than that I would have replaced the `.replace` with `sprintf` in this PR.